packages/@lwc/engine-core/src/framework/base-lightning-element.ts

```diff
@@ -490,20 +511,20 @@ const childGetters = [
     'lastElementChild',
 ] as const;
 
-function getChildGetter(methodName: typeof childGetters[number]) {
+function getChildGetter(methodName: typeof childGetters[number], renderer: RendererAPI) {
```

Comment:
@nolanlawson I believe this abstraction can be removed completely now... seems unnecessary since no more optimization is possible for those import statements.